lib/rules/sotd/cr-intro.js

```diff
@@ -0,0 +1,31 @@
+const self = {
+    name: 'sotd.cr-intro'
+,   section: 'document-status'
+,   rule: 'crIntro'
+};
+
+exports.name = self.name;
+
+exports.check = function (sr, done) {
```

Comment:
You are checking 2 paragraphs in that rule but the first one should be checked in 'lib/sotd/stability.js'. It's just a variation of that rule.